.eslintrc.js

```diff
@@ -436,7 +436,7 @@ module.exports = {
             ],
           },
         ],
-        'import/no-cycle': ['error', { ignoreExternal: true }],
+        'import/no-cycle': ['error', { ignoreExternal: false }],
```

Comment:
If the default is `false`, maybe it makes sense to just remove the options altogether?

Reply:
I've added a todo comment regarding this: https://github.com/mui/material-ui/pull/42705/commits/5e32a4c602fcffa92f2a52a0b088cf9a0c0cd3e5
Ideally, it should be `true` and hopefully it will be fixed at some point. 🤔 
WDYT?

Reply:
I think it depends on the impact of https://github.com/import-js/eslint-plugin-import/pull/2998
If after this change the difference between `true` and `false` is marginal, it could stay on `false`.

Reply:
I added a mention of this PR in the comment for more context.